include/lmms_basics.h

```diff
@@ -65,10 +63,6 @@ constexpr char LADSPA_PATH_SEPERATOR =
 #endif
 
 
-
-#define LMMS_STRINGIFY(s) LMMS_STR(s)
-#define LMMS_STR(PN)	#PN
```

Comment:
This original code has two macro definitions to ensure the argument is properly expanded, `PLUGIN_NAME` for example. Eliminating `LMMS_STR` will break macro expansions before stringification.

Reply:
So in this case, it's better to revert to the 2 macro definition at Plugin.h right? And the Pixmap file if it needs.

Reply:
Fixed.